packages/experimental/Divider/jest.config.js

```diff
@@ -0,0 +1,2 @@
+const { configureReactNativeJest } = require('@fluentui-react-native/scripts');
+module.exports = configureReactNativeJest('win32');
```

Comment:
configuring on win32 needs something extra as far i remember 🤔 . 

maybe adding  "@office-iss/react-native-win32": "^0.68.0", on your devDependencies , but would love to know if that need anything other config 👍

Reply:
Yeah I think you need to add RN win32 as a dev dependency to configure jest for win32.